lib/mock.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Janus.Mock do
+  @moduledoc false
+
+  @typedoc """
+  Tuple element containing request and response maps.
+
+  Response map should be compatible with formats handled by `Janus.Connection`, otherwise
+  it will not be handled by mentioned module and will crash `Janus.Connection` process.
+  """
+  @type request_response_pair :: {request :: map(), response :: map()}
+
+  @spec assert_pairs_shape([request_response_pair]) :: :ok
```

Comment:
I added this mostly because I caught myself making weird mistakes while writing tests. It helped me find the errors I made.